cmd/serverless/main.go

```diff
@@ -301,12 +306,51 @@ func runAgent(stopCh chan struct{}) (serverlessDaemon *daemon.Daemon, err error)
 		SubProcessor:         httpsecSubProcessor,
 	}
 
-	// start the experimental proxy if enabled
-	_ = proxy.Start(
-		"127.0.0.1:9000",
-		"127.0.0.1:9001",
-		serverlessDaemon.InvocationProcessor,
-	)
+	// NodeJS- and Python-specific support for AppSec: monitor the invocation through the runtime API monitoring until
+	// these tracers support the extension's universal instrumentation API
+	// TODO: remove the following block when the python and nodejs tracers support the extension universal instrumentation api
+	if rt := tags.GetRuntime(); appsecInstance != nil && (strings.Contains(rt, "nodejs") || strings.Contains(rt, "python")) {
```

Comment:
same here, could we extract the logic related to appsec in the appsec package and call it from here?

Reply:
Done in [5c782e7](https://github.com/DataDog/datadog-agent/pull/15378/commits/5c782e73f966af69709a01796f6c434fbb51c040)